onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc

```diff
@@ -345,6 +345,83 @@
         "^test_training_dropout_mask", // Runs but there's accuracy mismatch
         "^test_training_dropout_default" // Runs but there's accuracy mismatch
     ],
+    "current_failing_tests_OPENVINO_opset18": [
+        // pending opset 18 support, RUNTIME_EXCEPTION : Encountered unknown exception in Initialize()
```

Comment:
>/ pending opset 18 support, RUNTIME_EXCEPTION : Encountered unknown exception in Initialize() [](http://example.com/codeflow?start=9&length=93)

Does it mean that we are trying to update to a version that fails to initialize and run for a good number of tests in the same opset?

Reply:
Those tests are introduced in the Latest ONNX(1.13). They are not running in the old OpenVINO pipeline.
Scott did a similar thing in packaging pipeline https://github.com/microsoft/onnxruntime/pull/14259
Maybe they could be removed after the next upgrade of OpenVINO.

ONNX 1.12 used the protobuf 2.18.3, which doesn't meet the compliance.